ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-master__edge.yaml

```diff
@@ -62,6 +67,11 @@ test_binary_build_commands: |
   go mod vendor
   mkdir /.cache && chmod 775 -R /.cache ${GOPATH}
 tests:
+- as: build-multi-arch-images
```

Comment:
eventually it publishes the images, correct?
I'd rename it to ``publish-multi-arch-images``

Reply:
build and publish